src/rockraiders.rs

```diff
@@ -39,18 +31,3 @@ impl<'a, 'b> State<CustomGameData<'a, 'b>, StateEvent> for RockRaiders {
         Trans::None
     }
 }
-
-/// initialize the camera.
-fn initialize_camera(world: &mut World) {
-    let mut mat = Transform::default();
-    mat.move_global(Vector3::new(-2.0, 5.0, 2.0));
-    mat.yaw_global(Deg(-45.0));
-    mat.pitch_local(Deg(-45.0));
-
-    world
-        .create_entity()
-        .with(Camera::from(Projection::perspective(1.0, Deg(60.0))))
-        .with(mat)
-        .with(GlobalTransform::default())
-        .build();
```

Comment:
why could you remove all of this?

Reply:
because its the camera of the main state, where no camera is needed so far